### PR TITLE
Expose Style.localizeLabels API.

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

File: android/src/main/kotlin/com/mapbox/maps/mapbox_maps/StyleController.kt [hunk 1]
```diff
@@ -584,6 +586,17 @@ class StyleController(private val mapboxMap: MapboxMap) : FLTMapInterfaces.Style
     }
   }
 
+  override fun localizeLabels(
+    locale: String,
+    layerIds: MutableList<String>?,
+    result: FLTMapInterfaces.Result<Void>?
+  ) {
+    mapboxMap.getStyle {
+      it.localizeLabels(Locale(locale), layerIds)
```
[the remark this answers is not in the document]
Comment (yunikkk) [reply android/src/main/kotlin/com/mapbox/maps/mapbox_maps/StyleController.kt:595]: Basically yes, it'll log an error like `E/Mapbox: [maps-android\LocalizationPluginImpl]: Locale: blabla is not supported.`

File: android/src/main/kotlin/com/mapbox/maps/mapbox_maps/StyleController.kt [hunk 2]
```diff
@@ -5,12 +5,14 @@ import android.graphics.BitmapFactory
...
 import com.mapbox.common.Logger
 import com.mapbox.maps.*
+import com.mapbox.maps.extension.localization.localizeLabels
 import com.mapbox.maps.extension.style.layers.properties.generated.ProjectionName
 import com.mapbox.maps.extension.style.projection.generated.Projection
 import com.mapbox.maps.extension.style.projection.generated.getProjection
 import com.mapbox.maps.extension.style.projection.generated.setProjection
 import com.mapbox.maps.pigeons.FLTMapInterfaces
 import java.nio.ByteBuffer
+import java.util.*
```
[the remark this answers is not in the document]
Comment (yunikkk) [reply android/src/main/kotlin/com/mapbox/maps/mapbox_maps/StyleController.kt]: Done

**Review by alexander-kulikovskii — APPROVED** [review]

File: android/build.gradle [hunk 3]
```diff
@@ -54,9 +54,11 @@ android {
     }
 }
 
-project.apply {
-    from("$rootDir/gradle/ktlint.gradle")
-    from("$rootDir/gradle/lint.gradle")
+if (file("$rootDir/gradle/ktlint.gradle").exists() && file("$rootDir/gradle/lint.gradle").exists()) {
```
Comment (alexander-kulikovskii) [inline android/build.gradle:57]: is it possible to apply them independently?

File: example/pubspec.lock [hunk 4]
```diff
@@ -0,0 +1,330 @@
+# Generated by pub
```
Comment (alexander-kulikovskii) [inline example/pubspec.lock:1]: should lock file be here?

File: android/build.gradle (hunk 3 above)
Comment (yunikkk) [reply android/build.gradle:57]: Not sure it's needed, linters are both needed for the development, but should be skipped for users

File: example/pubspec.lock (hunk 4 above)
Comment (yunikkk) [reply example/pubspec.lock:1]: Yes, for the example app

**Pull request merged**